cmd/info.go

```diff
@@ -0,0 +1,103 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/fatih/color"
+	"github.com/google/go-github/github"
+	"github.com/ovh/svfs/svfs"
+	"github.com/spf13/cobra"
+)
+
+var (
+	currentVersion = "v" + svfs.Version
+)
+
+const (
+	gitOwner = "OVH"
+	gitRepo  = "svfs"
+)
+
+// Print some SVFS information
+func printReleaseInfo(name, tag string, time *github.Timestamp, prerelease bool, err error) {
+	p := fmt.Printf
+
+	p("* Version name : %s\n", name)
+	p("* Tag name : %s\n", tag)
+	p("* Is pre-release : %t\n", prerelease)
+	if time != nil {
+		p("* Published at : %s\n", *time)
+		p("\n\n")
+	}
+	if err != nil {
+		color.Red("\nYour SVFS version is invalid ! Check error just bellow.")
+	}
+}
+
+// Get more information on GitHub API about the current release you have
+func getCurrentReleaseInfo(client *github.Client) (cName, cTag string, cTime *github.Timestamp, cPrerelease bool, err error) {
+	orgs, _, err := client.Repositories.GetReleaseByTag(gitOwner, gitRepo, currentVersion)
+	if err != nil {
+		return "?", "?", nil, false, err
+	}
+	c := orgs
+
+	return *(c.Name), *(c.TagName), c.PublishedAt, *(c.Prerelease), nil
+}
+
+// Get more information on GitHub API about the latest release
+func getLastReleaseInfo(client *github.Client) (lName, lTag string, lTime *github.Timestamp, lPrerelease bool, err error) {
+	opt := &github.ListOptions{PerPage: 1}
+	orgs, _, err := client.Repositories.ListReleases(gitOwner, gitRepo, opt)
+	if err != nil {
+		return "?", "?", nil, false, err
+	}
+
+	length := len(orgs) - 1
+	l := orgs[length]
+
+	return *(l.Name), *(l.TagName), l.PublishedAt, *(l.Prerelease), nil
+}
+
+// Represents the info command, lVar for lastest release, cVar for current release
+var infoCmd = &cobra.Command{
+	Use:   "info",
+	Short: "Display fs-wide information",
+	Long: `Display some information about SVFS current version and
```

Comment:
I tend to prefer using double quotes for multiline strings since it doesn't source code identation.
For instance : https://github.com/xlucas/euro2016/blob/master/cmd/root.go#L12.